auto_dev/constants.py

```diff
@@ -110,7 +110,9 @@ def main():
     "termination_abci": "bafybeieb3gnvjxxsh73g67m7rivzknwb63xu4qeagpkv7f4mqz33ecikem",
 }
 AGENT_PUBLISHED_SUCCESS_MSG = "Agent published successfully."
-
+CREATE_AGENT_WORKFLOW_PATH = os.path.join(
```

Comment:
We already have a dir for the repo, report gonna have a lot more and gonna be able to list the workflows, so maybe have a folder?